src/Codeception/Subscriber/Console.php

```diff
@@ -536,7 +536,9 @@ protected function detectWidth()
             }
         } elseif ($this->isWin() && (php_sapi_name() === "cli")) {
             exec('mode con', $output);
-            preg_match('/^ +.* +(\d+)$/', $output[4], $matches);
+            if (isset($output[4])) {
+                preg_match('/^ +.* +(\d+)$/', $output[4], $matches);
+            }
             if (!empty($matches[1])) {
```

Comment:
Actually, that makes sense logically, without regard to the notice or not for `$matches`.  I'll update it.